torchvision/utils.py

```diff
@@ -382,6 +389,116 @@ def draw_keypoints(
     return torch.from_numpy(np.array(img_to_draw)).permute(2, 0, 1).to(dtype=torch.uint8)
 
 
+# Flow visualization code adapted from https://github.com/tomrunia/OpticalFlow_Visualization
+@torch.no_grad()
+def flow_to_image(
+    flow: torch.Tensor,
+) -> torch.Tensor:
+
+    """
+    Converts the two-dimensional flow to an RGB Image.
+
+    Args:
+        flow (Tensor): Flow of shape (2, H, W) and dtype torch.float.
```

Comment:
On top of single flow inputs of shape `(2, H, W)`, I think it would be very useful to also support batches of flows of shape `(N, 2, H, W)`.

But I'm happy to leave that for a follow-up PR

Reply:
Other utilities don't support batched input. I would like to know why we need to support batches of flow in this case? (Probably since we are operating on videos?)
Will follow-up in another PR.

Reply:
The model outputs batched input. Imagine you have a video for which you want to get all predicted flows as images to re-construct a gif of predicted flow afterwards.

You'll first get batched predicted flow, then if `flow_to_image` does not support batches, you'll have to manually loop over it. This is both inconvenient and also probably slower. It'd be nice to just be able to do

```py
predicted_flows = model(...)[-1]   # N 2 H W
images = flow_to_image(predicted_flows)  # N 3 H W
```

It's the most common use-case and this is what we'll need when we write a gallery example as well